lm_eval/base.py

```diff
@@ -22,7 +23,10 @@ def __init__(self, stop_words=None, requires_execution=True):
         """
         self.stop_words = stop_words
         self.requires_execution = requires_execution
-        self.dataset = load_dataset(path=self.DATASET_PATH, name=self.DATASET_NAME)
+        try:
+            self.dataset = load_dataset(path=self.DATASET_PATH, name=self.DATASET_NAME)
+        except:
+            warn("No HF Dataset specified. Task instance must provide own dataset.")
```

Comment:
This warning may give the user the impression that something is wrong, and is more for contributors who add tasks. I suggest we tune it down to something like: `This task will use a locally downloaded dataset, not from the HF hub`, wdyt?